data/strings/types_strings.txt

```diff
@@ -13930,6 +14205,7 @@
     en = Cairn
     de = Steinhaufen
     et = Karjäär
+    hi = केअर्न
```

Comment:
```suggestion
    hi = स्तूप|टीला
```

Reply:
I had to drop टीला, cairns aren't searchable yet.